src/jit/compiler.cpp

```diff
@@ -7202,32 +7202,30 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
-    if (fp == nullptr)
+    if (FILE* fp = _wfopen(jitTimeLogCsv, W("a")))
```

Comment:
This is a bit too fancy, and doesn't follow the convention of explicitly comparing non-bools in an 'if'. Why not just:
```
FILE* fp = _wfopen(jitTimeLogCsv, W("a"));
if (fp != nullptr)
{
```